pkg/maintainer/spv/spv.go

```diff
@@ -49,18 +57,384 @@ func (sm *spvMaintainer) startControlLoop(ctx context.Context) {
 		}
 
 		select {
-		case <-time.After(sm.config.RestartBackOffTime):
+		case <-time.After(sm.config.RestartBackoffTime):
 		case <-ctx.Done():
 			return
 		}
 	}
 }
 
 func (sm *spvMaintainer) maintainSpv(ctx context.Context) error {
-	logger.Infof("Maintaining SPV proof...")
+	for {
+		if err := sm.proveDepositSweepTransactions(); err != nil {
+			return fmt.Errorf(
+				"error while proving deposit sweep transactions: [%v]",
+				err,
+			)
+		}
```

Comment:
Let's align it with how we log the activity for the wallet coordinator maintainer:

```suggestion
		logger.Infof("starting deposit sweep proof task execution...")

		if err := sm.proveDepositSweepTransactions(); err != nil {
			return fmt.Errorf(
				"error while proving deposit sweep transactions: [%v]",
				err,
			)
		}

		logger.Infof(
			"deposit sweep proof task run completed; next run in [%s]",
			sm.config.IdleBackOffTime,
		)
```

Compare:

https://github.com/keep-network/keep-core/blob/fc1bbf3974f4562f65d74208f11c976fc04fe0d7/pkg/maintainer/wallet/wallet.go#L69-L96

Reply:
Done in 61e77e4760e35d764dd85340f0251116f7ccc49f.